### ISSUE TITLE
Group Discussion Links - Switch to discussion not working

### ISSUE THREAD

**Issue by stephenlevitt** [jamesjonesmath/canvancement#46]
I was unable to get the switch to discussion part to work although the rest of the script works fine for me.

The issue was
`const groupLinks = document.querySelectorAll('div#left-side > ul > li a');`
resulting in 0 being returned for `groupLinks.length`

Changing the above line to:
`const groupLinks = document.querySelectorAll('div#sticky-container > ul > li a');`
solved the problem.

Thanks for a very useful script!

**Comment by jamesjonesmath** [comment]
Thanks @stephenlevitt  for pointing this out and supplying the fix. 

I did some additional cleanup, refactoring, and lint checking, so the changes are substantial, even though little has changed. I also cleaned up some of the instructions, that were off.

Canvas is now, at least for me, correctly displaying the name of the course, so I removed the code to download that and replace it. It might have been a little too ambitious on my part, so if it creates any problems, let me know.
